tests/com/google/common/css/compiler/gssfunctions/GssFunctionsTest.java

```diff
@@ -204,6 +204,54 @@ public void testMakeTranslucent_otherLocale() {
     }
   }
 
+  public void testSaturateColor() throws GssFunctionException {
```

Comment:
Can you add a test or two that exercises the wrap-around behavior of hue and the clipping behavior of saturation and lightness?